src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/add_product_row.html.twig

```diff
@@ -59,7 +59,16 @@
   <td id="addProductTotalPrice"></td>
   {% if orderForViewing.hasInvoice() %}
     <td class="addProductInvoice pr-2">
-      {{ form_row(addProductRowForm.invoice) }}
+      {{ form_row(addProductRowForm.invoice, {
+        'attr': {
+          'data-modal-translations':        {
+            'modal.title': 'Create a new invoice'|trans({}, 'Admin.Orderscustomers.Feature'),
+            'modal.apply': 'Create'|trans({}, 'Admin.Actions'),
+            'modal.cancel': 'Cancel'|trans({}, 'Admin.Actions'),
+            'modal.body': 'Are you sure you want to create a new invoice?'|trans({}, 'Admin.Orderscustomers.Notification')
+          }|json_encode|replace("'", "&#39;")|raw|trim
```

Comment:
Not a good idea to use json encode here.
Use as many data attributes as there are.
This, plus the raw isn't a good idea.

Reply:
@zuk3975 so as modified in this PR https://github.com/PrestaShop/PrestaShop/pull/18690
Seems like the `|replace("'", "&#39;")|raw` is useless and potentially introduce security breach So you should be able to remove it

Reply:
okay